Dockerfile_RISCV

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+RUN apt-get update 
+RUN apt-get install -y build-essential git curl gawk texinfo expat \
+    libexpat1-dev bison flex libz-dev python autoconf
+# Prepare to download and compile RISCV toolchain
+ENV HOME_DIR=/home/root
+ENV RISCV_SRC_DIR=$HOME_DIR/riscv-gnu-toolchain
+ENV RISCV_DIR=$HOME_DIR/riscv-tools
+ENV mkdir -p $HOME_DIR
+WORKDIR $HOME_DIR
+RUN git clone --recursive https://github.com/riscv/riscv-gnu-toolchain -j 8
```

Comment:
Another option is to replace `-j 8` with `-j$(nproc)` may let this build faster on large machines without exploding normal ones.